docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -1,8 +1,215 @@
 # BigDL-Nano PyTorch Inference Overview
 
-add a link for examples here.
+BigDL-Nano provides serveral APIs which can easily apply optimizations on inference pipeline to improve latency and througput. Currently, performance accelerations are achieved by integrating extra runtime as inference backend engine or using quantization methods on full-precision trained models to reduce computation during inference. Trainer (`bigdl.nano.pytorch.Trainer`) provides the APIs for all optimizations you need for inference.
 
-###  Runtime Acceleration
-onnx runtime, openvino
+For runtime acceleration, BigDL-Nano has enabled two kinds of runtime for users in `Trainer.trace()`, ONNXRuntime and OpenVINO.
 
-### Quantization
\ No newline at end of file
+For quantization, BigDL-Nano provides only post-training quantization `trainer.quantize()` for users to inference with models of 8-bit precision. Quantization-Aware Training is not available for now. Model conversion to 16-bit like BF16, FP16 will be coming soon.
+
+Before you go ahead with these APIs, you have to make sure BigDL-Nano is correctly installed for pytorch. If not, please follow [this](../Overview/nano.md) to setup your environment.
+
+##  Runtime Acceleration
+All available runtime accelerations are integrated in `Trainer.trace(accelerator='onnxruntime'/'openvino')` with different accelerator value. Before you know about BigDL-Nano and any optimizations on inference, taking mobilenetv3 as example, you may have one script for training and inference like this:
+```python
+from torchvision.models.mobilenetv3 import mobilenet_v3_small
+import torch
+from torch.utils.data.dataset import TensorDataset
+from torch.utils.data.dataloader import DataLoader
+from bigdl.nano.pytorch import Trainer
+# step 1: create your model
+model = mobilenet_v3_small(num_classes=10)
+
+# step 2: prepare your data and dataloader
+x = torch.rand((10, 3, 256, 256))
+y = torch.ones((10, ), dtype=torch.long)
+ds = TensorDataset(x, y)
+dataloader = DataLoader(ds, batch_size=2)
+
+# (Optional) step 3: Something else, like training ...
+trainer = Trainer()
+trainer.fit(model, dataloader)
+...
+...
+
+# Inference/Prediction
+trainer.validate(ort_model, dataloader)
+trainer.test(ort_model, dataloader)
+trainer.predict(ort_model, dataloader)
+```
+### ONNXRuntime Acceleration
+Before you start with any BigDL-Nano usage, you are required to install some onnx packages as follow to setup your environment with ONNXRuntime acceleration.
```

Comment:
"Before you start with any BigDL-Nano usage" might be confusing. How about "Before you start with onnxruntime accelerator"?